app/javascript/pages/Dashboard/index.js

```diff
@@ -141,12 +130,12 @@ const LeaderboardContainer = ({
         <div className={s.topVolunteers}>Top Volunteers</div>
         <Filter
           collection={offices}
-          value={dashboardOfficeFilter.value}
+          value={dashboardOfficeFilter.value === 'current' ? currentUser.office.id : dashboardOfficeFilter.value}
           onChange={(_event, _index, value) => changeDashboardOfficeFilter(value)}
           itemValueProp="name"
         />
       </div>
-      {topUsers(dashboardOfficeFilter, users).map((user, i) => (
```

Comment:
This doesn't seem to be needed since it's already sorted by hours in the server here: https://github.com/zendesk/volunteer_portal/blob/master/app/graph/resolvers/user_resolver.rb#L92